modules/bzip2/1.0.8/patches/add_build_file.patch

```diff
@@ -0,0 +1,36 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,33 @@
++cc_binary(
++    name = "bzip2",
++    srcs = ["bzip2.c"],
++    copts = [
++        "-D_FILE_OFFSET_BITS=64",
++    ],
++    deps = [
++        ":bz2",
++    ],
++    visibility = ["//visibility:public"],
++)
++
++cc_library(
++    name = "bz2",
++    srcs = [
++        "blocksort.c",
++        "bzlib.c",
++        "bzlib_private.h",
++        "compress.c",
++        "crctable.c",
++        "decompress.c",
++        "huffman.c",
++        "randtable.c",
++    ],
++    hdrs = [
++        "bzlib.h",
++    ],
++    copts = [
++        "-D_FILE_OFFSET_BITS=64",
++    ],
++    strip_include_prefix = "",
```

Comment:
Does an empty `strip_include_prefix` have an effect here?

Reply:
Good question - this came from the dropbox dbx_build_tools repo originally.

Reply:
Turns out it does - its working around the fact that bzip2 internally uses <> includes